person.rb

```diff
@@ -0,0 +1,48 @@
+require '/decorate'
+
+# parent class Person.new(age, name, parent_permision)
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
```

Comment:
- Kindly fix the issues for importing or require the file functionalities to be used or accessed by another file. The project is not running as expected, please fix it.

![image](https://user-images.githubusercontent.com/55635977/187002302-b8de2118-06b0-4c35-a1be-5422194b3b67.png)